libmachine/drivers/base_test.go

```diff
@@ -0,0 +1,31 @@
+package drivers
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestIP(t *testing.T) {
+	ip, err := (&BaseDriver{}).GetIP()
```

Comment:
```
assert.Equal(t, ip, c.expectedIp)
assert.Equal(t, err, c.expectedErr)
```

should really be:

```
assert.Equal(t, c.expectedIp, ip)
assert.Equal(t, c.expectedErr, err)
```

Otherwise, in case of test failure, the error message will be confusing